README.md

```diff
@@ -186,9 +183,7 @@ See our [roadmap](#future-work) for details.
 ## Known Issues
 
 There are a few bugs in Zebra that we're still working on fixing:
-- [Zebra's address book can use all available memory #1873](https://github.com/ZcashFoundation/zebra/issues/1873)
 - [Zebra does not evict pre-upgrade peers from the peer set across a network upgrade #706](https://github.com/ZcashFoundation/zebra/issues/706)
```

Comment:
Our alternative implementation was documented in PR #3181, so we can delete this known issue:

```suggestion
```